src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -1235,4 +1252,19 @@ private int getMaxQueueItemCountIndex(String limit) {
         }
         return defaultMaxIndex;
     }
+
+    /**
+     * find first matching Queue-Configuration
+     * @param queueName search first configuration for that queue-name
+     * @return null when no queueConfiguration's RegEx matches given queueName - else the QueueConfiguration
+     */
+    private QueueConfiguration findQueueConfiguration(String queueName) {
+        // find a retry interval from the queue configurations
+        for (QueueConfiguration queueConfiguration : queueConfigurations) {
```

Comment:
Why do we state "_retry interval_" in comment? Seems that code has nothing to do with that directly.

Reply:
Ah - I extracted this code-fragment to a method.
Code was originally part of "de-queue slowdown feature". Will fix